lib/matplotlib/rcsetup.py

```diff
@@ -420,6 +420,13 @@ def validate_aspect(s):
         raise ValueError('not a valid aspect specification')
 
 
+def validate_fontsize_None(s):
+    if s is None or s == 'None':
```

Comment:
Should `s == 'none'` be acceptable too?

Reply:
I just copied `float_or_None`; I think this is here because when you type "None" in the matplotlibrc it comes back as a string.  So I'd argue that should be "None" and not allow "none".